### PR TITLE
Update README.md for branch-0.5

### PR DESCRIPTION
**Pull request by zhixingheyi-tian**
## What changes were proposed in this pull request?

Update README.md for branch-0.5
Add performance data;
Add contributors.

## How was this patch tested?

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,23 +106,45 @@ Compression Codec - Choose compression type for OAP data files.
...
+Cases:                   | T1/ms | T2/ms | T3/ms | Median/ms 
+---------------------    | ----- | ----- | ----- | ---------
+oap                      |  4241 |  2074 |  2014 |     2074
+oap-with-index,cache     |  1141 |   922 |   503 |      922  
+Orc                      | 20121 | 22455 | 20701 |    20701 
+Orc-with-index           | 18089 | 19127 | 18500 |    18500 
+parquet                  | 18116 | 16929 | 19551 |    18116  
+parquet-with-index       | 18104 | 18984 | 17966 |    18104  
+parquet-with-cache       |  3234 |  1586 |  1628 |     1628  
+parquet-with-cache,index |  3244 |  1229 |   858 |     1229
```
Comment (LuciferYang) [inline README.md]: Have page caches been cleaned up between each query?

Comment (zhixingheyi-tian) [reply README.md]: @LuciferYang 
No

File: README.md [hunk 2]
```diff
@@ -106,23 +106,45 @@ Compression Codec - Choose compression type for OAP data files.
...
+Orc                      | 20121 | 22455 | 20701 |    20701 
+Orc-with-index           | 18089 | 19127 | 18500 |    18500 
+parquet                  | 18116 | 16929 | 19551 |    18116  
+parquet-with-index       | 18104 | 18984 | 17966 |    18104  
+parquet-with-cache       |  3234 |  1586 |  1628 |     1628  
+parquet-with-cache,index |  3244 |  1229 |   858 |     1229  
+
+
+## Contributors
+@adrian-wang (Daoyuan Wang, Intel)
```
Comment (LuciferYang) [inline README.md]: @adrian-wang and @chenghao-intel no longer in Intel?

File: README.md [hunk 3]
```diff
@@ -106,23 +106,45 @@ Compression Codec - Choose compression type for OAP data files.
...
+@ConeyLiu (Xianyang Liu, Intel)
+@gczsjdy (Chenzhao Guo, Intel)
+@jerrychenhf (Chen Haifeng, Intel)
+@JkSelf (Ke Jia, Intel)
+@lee-lei (Lei Li, Intel)
+@linhong-intel (Linhong Liu, Baidu)
+@LuciferYang (Jie Yang, Baidu)
+@Moonlit-Sailor (Yi Cui, Alibaba Group)
+@songzhan01 (Zhan Song, Baidu)
+@TigerSong (XianZheng Song, Baidu)
```
Comment (LuciferYang) [inline README.md]: @TigerSong already leave Baidu now, may ANT FINANCIAL

**Comment by LuciferYang** [comment]
#989  do Typo Fixes, should we merge to branch-0.5?

File: README.md (hunk 1 above)
Comment (LuciferYang) [reply README.md]: Maybe we should clean it between each query, otherwise, the result is inaccurate.

File: README.md [hunk 4]
```diff
@@ -106,23 +106,45 @@ Compression Codec - Choose compression type for OAP data files.
...
+@zhangweilst (Wei Zhang, Shanghai Jiaotong University)
+@xuanyuanking (Yuanjian Li, Baidu)
+@yueawang (Yue Wang, Intel)
```
Comment (LuciferYang) [inline README.md]: Yue Wang in intel now?

File: README.md [hunk 5]
```diff
@@ -106,23 +106,45 @@ Compression Codec - Choose compression type for OAP data files.
...
+oap-with-index,cache     |  1141 |   922 |   503 |      922  
+Orc                      | 20121 | 22455 | 20701 |    20701 
+Orc-with-index           | 18089 | 19127 | 18500 |    18500 
+parquet                  | 18116 | 16929 | 19551 |    18116  
+parquet-with-index       | 18104 | 18984 | 17966 |    18104  
+parquet-with-cache       |  3234 |  1586 |  1628 |     1628  
+parquet-with-cache,index |  3244 |  1229 |   858 |     1229  
+
+
+## Contributors
```
Comment (LuciferYang) [inline README.md]: from commit log of 0.4->0.5, following users are contributors:
carsonwang
lee-lei
gczsjdy
songzhan01
JkSelf
LuciferYang
zhixingheyi-tian
ConeyLiu
LinhongLiu
WinkerDu

Comment (LuciferYang) [reply README.md]: WinkerDu report #930 and commit #936 .

**Comment by LuciferYang** [comment]
If we need pick #989  to branch-0.5, pizzaz93 is a new contributors

**Comment by LuciferYang** [comment]
If we need cherry-pick #989  to branch-0.5, pizzaz93 is a new contributors

File: README.md (hunk 1 above)
Comment (LuciferYang) [reply README.md]: Use BTree index or Bitmap index? Tested on TPCDS, Bitmap should be better than BTree based on past experience.

Comment (LuciferYang) [reply README.md]: From this table, I don't see any obvious advantage in using index, so should we remove this table?

Comment (zhixingheyi-tian) [reply README.md]: @LuciferYang 
In the experiment:
With cache conf, executor memory + offheap is close to total memory of node.
When disable Oap cache, we don't  clean up Page cache.

In Readme, it is reasonable and more convincing to do so, and can't ask users to actively clean up the page cache.
"over 10x boost" is only for these simple query.

While in benchmark report, it is necessary to design various cases  in detail, and workload can't be these simple queries.

Thanks.

File: README.md [hunk 6]
```diff
@@ -106,23 +106,45 @@ Compression Codec - Choose compression type for OAP data files.
...
-
-2. "SELECT * FROM store_sales WHERE ss_ticket_number < 10000 AND ss_net_paid BETWEEN 100.0 AND 110.0")
-
-Q12:                  | T1/ms | T2/ms | T3/ms | Median/ms 
---------------------- | ----- | ----- | ----- | ---------
-oap-with-index        |    509|   431 |   437 |      437
-parquet-with-index    |    944|   930 |  1318 |      944
-parquet-without-index |   2084|  1895 |  2007 |     2007
-
+Take 1 simple ad-hoc query as instance, the store_sales table comes from TPCDS with data scale 200G. Generally we can see over 10x boost in performance.
```
Comment (LuciferYang) [inline README.md]: From result table, I can't found 10x boost in performance in general.`oap-with-index,cache` has 2x~3x performance improvement over `oap`,  `Orc-with-index ` has 0.1x  performance improvement over `orc`, only `parquet-with-cache,index` or `parquet-with-cache` has 10x+  performance improvement over `parquet` :(

File: README.md (hunk 5 above)
Comment (zhixingheyi-tian) [reply README.md]: @LuciferYang 
What  is WinkerDu's full name in Baidu.

Comment (zhixingheyi-tian) [reply README.md]: @LuciferYang 
You have merged #989  to master. So also merge to  branch-0.5.

File: README.md (hunk 2 above)
Comment (adrian-wang) [reply README.md]: @ chenghao-intel SlideTalk.com :)

File: README.md (hunk 5 above)
Comment (LuciferYang) [reply README.md]: WinkerDu is Ripeng Du

**Comment by LuciferYang** [comment]
@zhixingheyi-tian 
```
You have merged #989 to master. So also merge to branch-0.5.
```
done.

File: README.md (hunk 2 above)
Comment (LuciferYang) [reply README.md]: @adrian-wang SlideTalke.com tell me : internal error - server connection terminated, haha

File: README.md (hunk 6 above)
Comment (LuciferYang) [reply README.md]:
```
I will suggest adding benchamark report to the "wiki" later.
```
excellent~

File: README.md (hunk 5 above)
Comment (adrian-wang) [reply README.md]: IMHO, We should keep version-specific contributors in RELEASE NOTE instead of project README.

File: README.md (hunk 1 above)
Comment (zhixingheyi-tian) [reply README.md]: @JkSelf 
You are right. Currently this test run on oap-perf-suite using dropCache() after each query.
Now ,all sample queries of oap-perf-suite Can not reflect the benefits of index performance.

Later, We can select some complex workloads to test, and add the benchmark report to the "wiki".

File: README.md (hunk 5 above)
Comment (zhixingheyi-tian) [reply README.md]: @adrian-wang 

Release note always have contributors.

**Pull request merged**